src/test/java/com/limechain/runtime/RuntimeBuilderTest.java

```diff
@@ -0,0 +1,30 @@
+package com.limechain.runtime;
+
+import com.limechain.rpc.server.AppBean;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.mockito.MockedStatic;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mockStatic;
+
+class RuntimeBuilderTest {
+
+    @Test
+    @Disabled("Very lengthy, needs either refactoring in the code being tested or a lot of effort into mocking")
+    void buildsRuntimeWithVersionInCustomSection() throws IOException {
```

Comment:
We might as well delete this test, since it'll be a cumbersome task to write properly.
Main issue is the dependency on the spring context which we have in the very constructor of a `Runtime` instance.
A few approach options are possible:
Ideally, we'd want an end-to-end test with the real blob, so we could:
1) try to mock out the entire `HostApi` implementations;
2) try to stub with an empty implementation of all imports;
3) give up on the end to end / integration test and simply test what's relevant to the `RuntimeBuilder` (which is in sync with unit testing, but we lose the actual check that our logic works with a real blob)

The frustrating part is that the currently written logic actually works, but for now only inspected whilst debugging in production... some kind of automation and a regression test that we won't break it later would be very fit at this point.

Reply:
Have you tried using the runtime that's in the examples folder of wasmer-java? It is very lightweight and only requires 2 function imports and a memory import.

Reply:
In a side discussion, we decided to scrape the test for now as out of scope. Added to `TODO.md` instead.